example/vault-client-namespace-role-binding.yaml

```diff
@@ -0,0 +1,14 @@
+# Used by vault pki to provide CA to client namespaces
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: vault-namespace-admin
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: admin
+subjects:
+  - kind: ServiceAccount
+    name: vault-pki
+    # Placeholder, patch with the client namespace value
+    namespace: example
```

Comment:
You only need to specify this if  the pki lives in a different namespace to vault?

Reply:
no, that is needed in the namespaces where clients live, so vault-pki can create the CA secret (or potentially configmap)